ironic/drivers/modules/drac/raid.py

```diff
@@ -645,18 +682,18 @@ def _assign_disks_to_volume(logical_disks, physical_disks_by_type,
          interface_type, size_mb), disks in physical_disks_by_type.items():
 
         if ('disk_type' in logical_disk
-            and logical_disk['disk_type'] != disk_type):
+                and logical_disk['disk_type'] != disk_type):
             continue
 
         if ('interface_type' in logical_disk
-            and logical_disk['interface_type'] != interface_type):
+                and logical_disk['interface_type'] != interface_type):
             continue
 
         # filter out disks without free disk space
         disks = [disk for disk in disks if free_space_mb[disk] > 0]
 
-        # sort disks by free size which is important if we have max disks limit
-        # on a volume
```

Comment:
Upstream reviewers will probably ask you to revert this change since it's outside the scope of this patch.

Reply:
Done